scss/_alert.scss

```diff
@@ -3,13 +3,24 @@
 //
 
 .alert {
+  // scss-docs-start alert-css-vars
+  --#{$variable-prefix}alert-bg: transparent;
+  --#{$variable-prefix}alert-padding: #{$alert-padding-y $alert-padding-x};
+  --#{$variable-prefix}alert-margin-bottom: #{$alert-margin-bottom};
+  --#{$variable-prefix}alert-color: #{$alert-padding-y $alert-padding-x};
```

Comment:
That's a great color ! 😂 + not used

Reply:
Yep :) It is tackled directly in Bootstrap via https://github.com/twbs/bootstrap/commit/a2c056e1f6d5925a5e6fef1f5d709f07a2a535d1